.test_database_urls/mysql_5_7

```diff
@@ -1,2 +1,2 @@
 export TEST_DATABASE_URL="mysql://root:prisma@localhost:3306"
-set -e TEST_SHADOW_DATABASE_URL
+unset TEST_SHADOW_DATABASE_URL
```

Comment:
Not a standard command :(

Damn, how do we solve this for all? Something like `if term is fish then set -e, else unset`?

Reply:
`export TEST_SHADOW_DATABASE_URL=''` ? or maybe we can find a way to test vitess without a separate instance for shadow databases

`set -e` makes my zsh crash.

Reply:
On zsh `set -e` returns value `1` if the variable is not set. On fish `unset` will crash due to command not existing. This means we cannot have these test files how they are now...

Reply:
Nope, we have to find a generic solution. I'll try to find some time to solve that today.